Assets/MRTK/Tools/BuildWindow/Arm64Workaround.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// This file contains a build post processor that adds a specific compiler flag to avoid
+// the ARM64 compiler issue described in this issue:
+// https://github.com/microsoft/MixedRealityToolkit-Unity/issues/7624
+// It works by updating the specific .vxcproj file and only modifying the ARM64 compiler
+// option to turn off the specific optmization that leads to the ARM64 issue
+
+// This build post processor should only after during UWP player build.
```

Comment:
I'm not sure exactly what this comment is intended to say, but maybe:

```suggestion
// This build post processor should only run after a UWP player build.
```